pack/tsk/tskp.json

```diff
@@ -395,6 +509,181 @@
+        "customization_text": "□ <b>Pessimistic Outlook.</b> Add the following criteria:\"you run out of cards in your hand\".\n□ <b>Trial and Error.</b> Add the following criteria: \"you are dealt damage or horror.\"\n□ <b>Independent Variable.</b> Add the following criteria: \"you discard a treachery or enemy from play.\"\n□ <b>Field Research.</b> Add the following criteria: \"you enter a location with 3 or more shroud.\"\n□□ <b>Peer Review.</b> The chosen criteria is met if any investigator at your location meets it, instead of only you. Other investigators at your location may trigger [fast] abilities on Empirical Hypothesis.\n□□ <b>Research Grant.</b> Empirical Hypothesis gains: \"[fast] Spend 2 evidence: Reduce the cost of the next card you play by 3.\"\n□□□ <b>Irrefutable Proof.</b> Empirical Hypothesis gains: \"[fast] Spend 3 evidence: Discover 1 clue at your location.\"\n□□□□ <b>Alternative Hypothesis.</b> After you exhaust Empirical Hypothesis, you may resolve its forced effect, choosing a criteria you have not chosen this round. Then, ready it.",
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "illustrator": "Isuardi Therianto",
+        "name": "Empirical Hypothesis",
+        "pack_code": "tskp",
+        "position": 41,
+        "quantity": 2,
+        "skill_intellect": 1,
+        "text": "Customizable. Limit 1 per investigator.\n<b>Forced</b> - At the start of the round: Choose one of the following criteria for this round -\n- You fail a test by 2 or more.\n- You succeed at a test by 3 or more.\nWhen the chosen criteria is met, you may exhaust Empirical Hypothesis to add 1 evidence to it.\n[fast] Spend 1 evidence: Draw 1 card.",
+        "traits": "Talent. Science.",
+        "type_code": "asset",
+        "xp": 0
+    },
+    {
+        "code": "09042",
+        "cost": 3,
+        "customization_options": [
+            {
+                "xp": 0,
+                "text_change": "insert",
+                "position": 0,
+                "choice": "choose_card",
+                "quantity": 1,
+                "card": {
+                    "type": ["asset"],
+                    "trait": ["Tome", "Spell"]
+                }
+            },
+            {
+                "xp": 1,
+                "text_change": "append"
+            },
+            {
+                "xp": 1,
+                "text_change": "append"
+            },
+            {
+                "xp": 2,
+                "text_change": "append"
+            },
+            {
+                "xp": 2,
+                "text_change": "insert",
+                "position": 0,
+                "choice": "choose_card",
+                "quantity": 2,
+                "card": {
+                    "type": ["asset"],
+                    "trait": ["Tome", "Spell"]
+                }
+            },
```

Comment:
You've missed 'Interwoven Ink' in the two sections below this, and here you've got 2 1xp options, 2 2xp options, a 3xp and a 4xp option. But the Quill has **three** 2xp options. So at this point you need to add one for Energy Sap, I believe. Text is: "<b>Energy Sap.</b> The Raven Quill gains: \"[fast] Exhaust The Raven Quill: Move 1 secret or charge from an asset you control to attached asset.\"" <- for your reference. 

I think this should be 
`"xp:" 2,
"text_change": "append"`

If I've understood correctly.